cmd/devices-api/api.go

```diff
@@ -92,7 +93,7 @@ func startWebAPI(logger zerolog.Logger, settings *config.Settings, pdb db.Store,
 	ddIntSvc := services.NewDeviceDefinitionIntegrationService(pdb.DBS, settings)
 	ddSvc := services.NewDeviceDefinitionService(pdb.DBS, &logger, settings)
 	ddaSvc := services.NewDeviceDataService(settings.DeviceDataGRPCAddr, &logger)
-
+	ipfsSvc := ipfs.NewLoader(settings)
```

Comment:
Also, does "loader" make sense to you? I tend to assume that loaders are for loading-out-of.

Reply:
updated this to `NewGateway` that has an `UploadImage` method... maybe thats worse. open to other ideas!

Reply:
That might be a bit too broad.